### PR TITLE
Changed Colors and Styles

### PR DESCRIPTION
**Pull request by hariharan-mahadevan-10884**
Changed background colors, radio button styles, Table colors and Submit button style,

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -195,6 +196,22 @@ td, th {
...
+    border-radius:4px;
+    width:250px;
+    padding: 16px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 20px;
+    margin: 4px 2px;
+    -webkit-transition-duration: 0.4s; /* Safari */
+    transition-duration: 0.4s;
```
Comment (sivaraam) [inline css/main.css]: What's the use of the transition-duration ? What are you transforming ?

File: css/main.css [hunk 2]
```diff
@@ -195,6 +196,22 @@ td, th {
...
 }
 
+input[type=submit]
+{
+    border: 2px;
+    border-radius:4px;
+    width:250px;
+    padding: 16px 32px;
+    text-align: center;
+    text-decoration: none;
```
Comment (sivaraam) [inline css/main.css]: The above two lines don't seem to have any effect, what's their intended use ?

File: css/main.css [hunk 3]
```diff
@@ -195,6 +196,22 @@ td, th {
   text-align: center;
 }
 
+input[type=submit]
+{
+    border: 2px;
+    border-radius:4px;
+    width:250px;
```
Comment (sivaraam) [inline css/main.css]: That seems too wide. 🤔

**Review by sivaraam — CHANGES_REQUESTED** [review]
Needs changes. See if you could convert all the `px` values into `em`s. The `em` are a better than pixels as they scale well.

File: css/main.css (hunk 1 above)
Comment (hariharan-mahadevan-10884) [reply css/main.css]: Actually I changed the color of the button on hover later removed it . For that only I have used transition-duration

File: css/main.css (hunk 2 above)
Comment (hariharan-mahadevan-10884) [reply css/main.css]: Which two lines?

File: css/light-styles.css [hunk 4]
```diff
@@ -37,21 +38,48 @@ select {
 }
 
 table {
-  border-color: #E24E42;
+  border:none;
```
Comment (sivaraam) [inline css/light-styles.css]: I remember telling you not to keep anything other than color and font-related rules in this file.

File: css/light-styles.css [hunk 5]
```diff
@@ -37,21 +38,48 @@ select {
...
 }
 
 table th, table tr {
-  border-color: #fff;
-  background-color: #0A1612;
+  border:none;
```
Comment (sivaraam) [inline css/light-styles.css]: Same here.

File: css/light-styles.css [hunk 6]
```diff
@@ -37,21 +38,48 @@ select {
...
 }
 
 table td {
-  border-color: #fff;
-  background-color: #1A2930;
+  border:none;
+  background-color: #07889B;
+}
+
+.zeroPadding
```
Comment (sivaraam) [inline css/light-styles.css:54]: Give a better name than that.

Comment (sivaraam) [reply css/light-styles.css:54]: And yes move it to the main.css file

File: css/main.css [hunk 7]
```diff
@@ -143,7 +143,6 @@ html, body {
   text-align: left;
   padding-top: 1em;
   margin-right: 1em;
-  transition: border 0.5s linear;
```
Comment (sivaraam) [inline css/main.css:146]: This was there for a reason. Bring it back

File: css/main.css [hunk 8]
```diff
@@ -160,30 +159,19 @@ html, body {
...
 /* Hide radio date selector for small screens */
-.date-radio-container {
+#date-radio {
   display: none;
 }
 
 /* Hide radio dept selector for small screens */
-.halls-radio-container {
+#halls-radio {
   display: none;
```
Comment (sivaraam) [inline css/main.css:168]: Reason for changing the above classes into IDs ?

File: css/main.css [hunk 9]
```diff
@@ -160,30 +159,19 @@ html, body {
...
 }
-
 .item-description {
   padding-bottom: 0.3em;
   display: block;
 }
 
-/* Hide the status table and book button until the user selects
```
Comment (sivaraam) [inline css/main.css:177]: Don't remove it. Comment it for the mean time.

File: css/main.css (hunk 2 above)
Comment (sivaraam) [reply css/main.css]: text-align and text-decoration properties

File: css/main.css [hunk 10]
```diff
@@ -208,22 +196,59 @@ td, th {
...
+{
+    border: 0.125em ;
+    border-radius:0.25em;
     text-align: center;
     text-decoration: none;
     display: inline-block;
-    font-size: 20px;
-    margin: 4px 2px;
+    font-size: 1.35em;
+    margin:0em;
```
Comment (sivaraam) [inline css/main.css]: I guess you could remove the margin property. It defaults to 0, I guess.

File: index.html [hunk 11]
```diff
@@ -15,7 +15,9 @@
         <link rel="stylesheet" href="css/light-styles.css">
         <link rel="stylesheet" href="css/radio-button.css">
         <link rel="stylesheet" href="css/radio-button-light-style.css">
-
+		<link rel="stylesheet" href="css/check-box.css">
+		<link rel="stylesheet" href="css/check-box-light-style.css">
+
```
Comment (sivaraam) [inline index.html]: Add `type` attributes to `link` elements

File: index.html [hunk 12]
```diff
@@ -41,26 +43,24 @@ <h1 class="title">Seminar Hall Booking</h1>
...
                         Mobile : select
-                        Higher resolution devices : (radio) button
+                        Higher resolution devoces : radio button
                       -->
-                      <div class="date-select-container">
-                        <select id="date-select" name="date-select">
-                          <option value="select"> Select </option>
+                      <div id="date-select">
+                        <select id="dates" name="date">
                           <option value="20">20 Thu</option>
```
Comment (sivaraam) [inline index.html:54]: The default value should be `Select` for a reason. Further don't change the `ID`s and `name` attributes they are meant to be what they are.

File: index.html [hunk 13]
```diff
@@ -97,7 +97,7 @@ <h1 class="title">Seminar Hall Booking</h1>
                         </div>
 
                         <div class="css-button date-button-col">
-                          <input type="radio" name="date-radio" id="date-5" value="">
+                          <input type="radio" name="date" id="date-5">
                           <div class="css-button-label-container">
```
Comment (sivaraam) [inline index.html:101]: Revert the names to their old values and don't change the name and ID's in the future. _They are meant to be what they are_.

File: index.html [hunk 14]
```diff
@@ -118,15 +118,8 @@ <h1 class="title">Seminar Hall Booking</h1>
...
-                        to be updated by JS
-                        Mobile : select
-                        Higher resolution devices : (radio) button
-                      -->
-                      <div class="halls-select-container">
-                        <select id="halls-select" name="hall-select">
-                          <option value="select">Select</option>
+                      <div id="halls-select">
+                        <select id="halls" name="hall">
                           <option value="cse">  CSE </option>
```
Comment (sivaraam) [inline index.html:123]: Don't remove comments and do changes as stated before

File: index.html [hunk 15]
```diff
@@ -206,8 +199,27 @@ <h1 class="title">Seminar Hall Booking</h1>
                     </div> <!-- item -->
                   </div> <!-- .form-item -->
                 </div>  <!-- .form-item-container -->
-
-                <div id="status-table-container" class="form-item-container">
```
Comment (sivaraam) [inline index.html:210]: Be careful while adding new items. You seemed to have removed the id of the table container. Bring it back.

File: index.html [hunk 16]
```diff
@@ -219,52 +231,148 @@ <h1 class="title">Seminar Hall Booking</h1>
                       <div class="column">
                         <table id="status-table-1">
                           <thead>
-                            <tr>
-                              <th> I </th>
-                              <th> II </th>
-                              <th> III </th>
-                              <th> IV </th>
-                            </tr>
```
Comment (sivaraam) [inline index.html:227]: The `th` elements should be a child of `tr` element

File: index.html [hunk 17]
```diff
@@ -219,52 +231,148 @@ <h1 class="title">Seminar Hall Booking</h1>
...
+							    <input type="checkbox" name="hour" id="hour-4">
+                                <div class="css-button-label-container">
+									<label id="hour-label-4" for="hour-4">
+										Book now
+									</label>
+                                </div>	
+								</div>
+								</div>
+ 							</td>
+							</tr>
```
Comment (sivaraam) [inline index.html:295]: Fix the white space issues. They look too odd.

File: index.html [hunk 18]
```diff
@@ -219,52 +231,148 @@ <h1 class="title">Seminar Hall Booking</h1>
...
+							    <input type="checkbox" name="hour" id="hour-8">
+                                <div class="css-button-label-container">
+									<label id="hour-label-8" for="hour-8">
+										Book now
+									</label>
+                                </div>	
+								</div>
+								</div>
+ 							</td>
+							</tr>
```
Comment (sivaraam) [inline index.html:364]: White space issues again

File: index.html [hunk 19]
```diff
@@ -219,52 +231,148 @@ <h1 class="title">Seminar Hall Booking</h1>
...
                           </tbody>
                         </table>
                       </div> <!-- column -->
                     </div> <!-- item -->
                   </div> <!-- .form-item -->
                 </div>  <!-- form-item-container -->
 
-                <div id="button-container" class="form-item-container">
+                <div class="form-item-container">
                   <div class="form-item button">
```
Comment (sivaraam) [inline index.html:373]: Don't touch the ID's

**Review by sivaraam — CHANGES_REQUESTED** [review]
There seems to be lots of changes than the title of the Pull request states. Keep changes focused to the title of Pull request. Create new PRs for separate changes. 

You seem to using tabs for indentation. Switch to spaces. See how to make your text editor to insert spaces when you hit tab. The behaviour should be 4spaces for a single tab.

**Comment by sivaraam** [comment]
@sirganeshv There seems to be changes in JS files. You better have a look.

File: css/main.css [hunk 20]
```diff
@@ -207,6 +196,59 @@ td, th {
...
 }
 
+/*=========================================================================
+  Style for submit button
+=========================================================================*/
+input[type=submit]
+{
+    border: 0.125em solid;
+    border-radius:0.25em;
+    width:9.375em;
```
Comment (sivaraam) [inline css/main.css:206]: As stated in previous review. Doesn't that seem too wide.

**Review by sivaraam — CHANGES_REQUESTED** [review]

**Comment by sivaraam** [comment]
A few things I missed,

1. Change the title and description of this PR to reflect what it changes. In future, create separate PRs for separate changes.
2. Create more commits. Don't try to put every change you do into a single commit. It clutters the purpose of version control
3. The subject selection doesn't look good.
screenshot from 2017-04-23 08-10-38

**Comment by sivaraam** [comment]
One more important thing, I remember telling you to give the theme that  **resembles** the one found at the [Sass homepage](http://sass-lang.com/). Change the colors so that they **resemble** the homepage.

**Pull request closed without merging**